File: src/main/webapp/resources/js/app/widget/Column.js
var Column = fw.create([BaseWidget],{
  

  setLabel : function(label){
	  this.label = label;
  },
  
  getLabel : function(){
	  return this.label;
  },
  
  setField : function(field){
	  this.field = field;
  },
  
  getField : function(){
	  return this.field;
  },
  
  setAutoSize:function(autoSize){
	  this.autoSize = autoSize;
  }
  
});

File: src/main/java/com/ff/app/tags/Column.java
package com.ff.app.tags;


public class Column extends BaseTag{
	private String label;
	private String field;
	private Boolean autoSize;
	
	public Boolean getAutoSize() {
		return autoSize;
	}
	public void setAutoSize(Boolean autoSize) {
		this.autoSize = autoSize;
	}
	public String getLabel() {
		return label;
	}
	public void setLabel(String label) {
		this.label = label;
	}
	public String getField() {
		return field;
	}
	public void setField(String field) {
		this.field = field;
	}
	
	
}
